doc/man1/openssl.pod

```diff
@@ -308,27 +316,51 @@ RMD-160 Digest
 
 =item B<sha>
 
-SHA Digest
+SHA-0 Digest
```

Comment:
SHA-0 was removed a while ago and openssl(1) doesn't recognize sha as option... I mean this should be omitted, not fixed.